source/guides/clear/mixer.rst

```diff
@@ -738,32 +727,90 @@ content.
 
    #builder.conf
 
-   #VERSION 1.0
+   #VERSION 1.2
 
    [Builder]
-     CERT = "/home/clr/mix/Swupd_Root.pem"
-     SERVER_STATE_DIR = "/home/clr/mix/update"
-     VERSIONS_PATH = "/home/clr/mix"
-     YUM_CONF = "/home/clr/mix/.yum-mix.conf"
+      CERT = "/home/clr/mix/Swupd_Root.pem"
+      SERVER_STATE_DIR = "/home/clr/mix/update"
+      VERSIONS_PATH = "/home/clr/mix"
+      YUM_CONF = "/home/clr/mix/.yum-mix.conf"
+
 
    [Swupd]
-     BUNDLE = "os-core-update"
-     CONTENTURL = "<URL where the content will be hosted>"
-     VERSIONURL = "<URL where the version of the mix will be hosted>"
+      BUNDLE = "os-core-update"
+      CONTENTURL = "<URL where the content will be hosted>"
+      VERSIONURL = "<URL where the version of the mix will be hosted>"
+      COMPRESSION = ["external-xz"]
+      UPSTREAM_BUNDLES_URL = "https://github.com/clearlinux/clr-bundles/archive/"
 
    [Server]
-     DEBUG_INFO_BANNED = "true"
-     DEBUG_INFO_LIB = "/usr/lib/debug"
-     DEBUG_INFO_SRC = "/usr/src/debug"
+      DEBUG_INFO_BANNED = "true"
+      DEBUG_INFO_LIB = "/usr/lib/debug"
+      DEBUG_INFO_SRC = "/usr/src/debug"
 
    [Mixer]
-     LOCAL_BUNDLE_DIR = "/home/clr/mix/local-bundles"
-     LOCAL_REPO_DIR = ""
-     LOCAL_RPM_DIR = ""
-     DOCKER_IMAGE_PATH = "clearlinux/mixer"
-
-Additional explanation of variables in :file:`builder.conf` is provided in Table
-1.
+      LOCAL_BUNDLE_DIR = "/home/clr/mix/local-bundles"
+      LOCAL_REPO_DIR = "/home/clr/mix/local-yum"
+      LOCAL_RPM_DIR = "/home/clr/mixx/local-rpms"
```

Comment:
Is **mixx** a typo? On Ln 754, why does the value
`LOCAL_RPM_DIR =  "/home/clr/mixx/local-rpms"` ?

Reply:
you're right, it should be "mix", not "mixx".

Reply:
Cool. See latest commit.